### DISCUSSION TITLE
💬 Created a discord server

### DISCUSSION THREAD

**Discussion by bahree** [teslamate-org/teslamate#198] [General]
I thought this might be helpful and created a [discord server](https://discord.gg/pByXZ9S) that we can use for discussions. Often there are things that are more ideas, and discussions and not necessarily a bug or such and Github, whilst awesome doesn't have the constructs to allow that.

And this is where we all can learn and help each other. 

The link again: https://discord.gg/pByXZ9S

**Comment by Sjorsa** [comment]
Has this server been removed or is discord being buggy? When I join it doesn't join the server...

**Reply by JakobLichterfeld** [reply]
https://github.com/teslamate-org/teslamate/discussions/3416#discussioncomment-7645291
